README.md

```diff
@@ -72,8 +72,8 @@ Now, pull up a browser and open `pros-docs/build/index.html`
 
 Now you can easily edit the `.rst`, and make new files. If you would like to, submit a PR to get those files changed on the website.
 
-#### Hard
-Modifying the tabs theming
+#### Theme Changes
+For more advanced users, if there is a bug in the theme, or *syntax highlighting*, for example. The tabs theming may be changed if you follow this guide. 

```

Comment:
hmm, i think it's better to just remove references to level of difficulty entirely, and that would include phrases like "advanced users."

~also, is there a reason "syntax highlighting" is italicized?~ `theme, or *syntax highlighting*` -> `theme (like the syntax highlighting, for instance)`

lastly, I don't think we need to specifically call out the tabs theme here, as "theme" should cover both. the distinction can be left to later where it matters